HW0/P4/P4.py

```diff
@@ -30,47 +30,97 @@
     # Load true trajectory and plot it
     # Normally, this data wouldn't be available in the real world
     #####################
+    s_true=np.loadtxt('P4_trajectory.txt',delimiter=',')
```

Comment:
Thanks for these suggestions! That is very helpful. I will make sure to use that next time I load the text files.